docs/guides/buttons.md

```diff
@@ -1,5 +1,5 @@
 ---
-description: Learn how to create Slash Commands on your Discord Bot using the Pycord library!
+description: Learn how to create Slash Commands on your Discord Bot using the EpikCord.py library!
```

Comment:
This can be improved to something like "Learn how to create slash commands with EpikCord.py!" as that is more shortened and makes more sense to me